src/packer.pkr.hcl

```
@@ -0,0 +1,131 @@
+variable "ami_regions" {
+  default     = []
+  description = "The list of AWS regions to copy the AMI to once it has been created. Example: [\"us-east-1\"]"
+  type        = list(string)
+}
+
+variable "build_region" {
+  default     = "us-east-1"
+  description = "The region in which to retrieve the base AMI from and build the new AMI."
+  type        = string
+}
+
+variable "build_region_kms" {
+  default     = "alias/cool-amis"
+  description = "The ID or ARN of the KMS key to use for AMI encryption."
+  type        = string
+}
+
+variable "is_prerelease" {
+  default     = false
+  description = "The pre-release status to use for the tags applied to the created AMI."
+  type        = bool
+}
+
+variable "region_kms_keys" {
+  default     = {}
+  description = "A map of regions to copy the created AMI to and the KMS keys to use for encryption in that region. The keys for this map must match the values provided to the aws_regions variable. Example: {\"us-east-1\": \"alias/example-kms\"}"
+  type        = map(string)
+}
+
+variable "release_tag" {
+  default     = ""
+  description = "The GitHub release tag to use for the tags applied to the created AMI."
+  type        = string
+}
+
+variable "release_url" {
+  default     = ""
+  description = "The GitHub release URL to use for the tags applied to the created AMI."
+  type        = string
+}
+
+variable "skip_create_ami" {
+  default     = false
+  description = "Indicate if Packer should not create the AMI."
+  type        = bool
+}
+
+data "amazon-ami" "debian_buster" {
+  filters = {
+    name                = "debian-10-amd64-*"
+    root-device-type    = "ebs"
+    virtualization-type = "hvm"
+  }
+  most_recent = true
+  owners      = ["136693071363"]
+  region      = "${var.build_region}"
```

Remove unnecessary characters:
```suggestion
  region      = var.build_region
```

---

I rebased to include this missed item from the merge so you should see this corrected now.

---

Thanks!